app/commands/incident.py

```diff
@@ -312,13 +311,13 @@ def submit(ack, view, say, body, client, logger):
     text = f":lapage: An incident report has been created at: {document_link}"
     say(text=text, channel=channel_id)
 
-    # Reminder to brief up
-    text = ":alphabet-yellow-question: Is this a `cybersecurity incident` (secret/data leak, account compromise, attack)? Please initiate the briefing process for CCCS and TBS OCIO Cyber. This just means we send a summary of the incident (or initial findings and updates if incident is ongoing) to cyberincident@cyber.gc.ca and CC zztbscybers@tbs-sct.gc.ca, and security@cds-snc.ca! CCCS will reach out with a case number, and any questions if they need more information."
-    say(text=text, channel=channel_id)
+    # # Reminder to brief up
+    # text = ":alphabet-yellow-question: Is this a `cybersecurity incident` (secret/data leak, account compromise, attack)? Please initiate the briefing process for CCCS and TBS OCIO Cyber. This just means we send a summary of the incident (or initial findings and updates if incident is ongoing) to cyberincident@cyber.gc.ca and CC zztbscybers@tbs-sct.gc.ca, and security@cds-snc.ca! CCCS will reach out with a case number, and any questions if they need more information."
+    # say(text=text, channel=channel_id)
```

Comment:
Favour to ask - can you remove the commented code?

Reply:
Thanks for catching this!! :grimace: